src/components/block/ProgramCard/ProgramCard.js

```diff
@@ -1,19 +1,27 @@
+import classNames from "classnames/bind";
 import PropTypes from "prop-types";
 import React from "react";
+
 import s from "./ProgramCard.module.css";
 
-const ProgramCard = ({ theme, image, imageAlt, title, description }) => (
-  <div
-    className={`${s.programCardContainer} 
-    ${theme === "light" ? s.light : s.dark}`}
-  >
-    <div>
-      <img className={s.img} src={image} alt={imageAlt} />
+const ProgramCard = ({ theme, image, imageAlt, title, description }) => {
+  const classStyles = classNames.bind(s);
```

Comment:
Regarding the `bind` way of doing this, it looks like in the [README](https://github.com/JedWatson/classnames/tree/v2.2.6#alternate-bind-version-for-css-modules) they recommend a different method if you're in an ES2015 (ES6) environment:

> _Note that in ES2015 environments, it may be better to use the "dynamic class names" approach documented above._

If you look at [the other section that they refer to](https://github.com/JedWatson/classnames/tree/v2.2.6#dynamic-class-names-with-es2015), then it suggests using the special bracket notation for using a variable's value as a key in an object literal, e.g. `{ [myKey]: "foo" }`. What this does is instead of creating an object with a key of `"myValue"`, it instead looks up the value that a variable named `myKey` has and uses _that_ as the actual key.

I think applied to `classnames` and CSS modules, they may be suggesting something like this:

```js
import s from "./Foo.module.css";

const className = classnames({
  [s.card]: true,
  [s.light]: theme === "light",
});
```